WNPRC_EHR/resources/schemas/dbscripts/postgresql/wnprc-22.009-22.010.sql

```diff
@@ -0,0 +1,12 @@
+
+-- OCONNOR fixes
+ALTER TABLE oconnor.inventory ADD COLUMN IF NOT EXISTS ModifiedBy USERID;
+ALTER TABLE oconnor.inventory ADD COLUMN IF NOT EXISTS CreatedBy USERID;
+ALTER TABLE oconnor.inventory_removed ADD COLUMN IF NOT EXISTS ModifiedBy USERID;
+ALTER TABLE oconnor.inventory_removed ADD COLUMN IF NOT EXISTS CreatedBy USERID;
+
+-- EHR fixes
+ALTER TABLE ehr.protocolprocedures ADD COLUMN IF NOT EXISTS remark TEXT;
```

Comment:
This is safer because WNPRC_EHR depends on EHR, but if in the future this table is removed or this column is deleted in an `ehr` script, we'll still run this script afterwards. I know it means another FB and PR in a separate repo, but I think we should err on the safe side and do this from the EHR module. However, we don't need to merge it forward, as we only care about it running on the WNPRC server.